src/main/java/net/dv8tion/jda/internal/interactions/CommandInteractionImpl.java

```diff
@@ -123,6 +125,19 @@ private void parseResolved(JDAImpl jda, DataObject resolveJson)
         }
     }
 
+    private void buildText()
+    {
+        //Get text like the text that appears when you hover over the interaction in discord
+        StringBuilder builder = new StringBuilder();
+        builder.append("/").append(getCommandPath().replace("/", " ")).append(" ");
```

Comment:
There is no reason to use getCommandPath here

Reply:
removed and replaced